README.md

```diff
@@ -29,17 +34,19 @@ Copy this row to add yourself
 
 # Timeline
 
-TODO: Add actual dates
-
-- Date 0: Engage with various community members to evangelize idea, identify contributors, get feedback, etc
-- Date 1: Align on the topics & learning goals
-- Date 2: Develop a core group of contributors
-- Date 3: Survey draft written
-- Date 4: Survey text finalized
-- Date 5: Survey launched
-- Date 6: Survey closed
-- Date 7: Survey raw data available
-- Date 8: Synthesis available
+We are working backwards from an April 2023 release date for the report.  We are hopeful that our proposed KubeCon talk on this survey will be accepted.
+
+- November 7 - 18: Engage with various community members to evangelize idea, identify contributors, get feedback, etc
+- November 22: Finalized list of topics & learning goals
+- December 9: First draft of survey text written
+- December 10 - 22: Collect feedback on the draft survey text
+- December 23 - January 5: Holiday break
+- January 13: Survey release candidate #1 available as a web link
```

Comment:
should we say how we want to distribute it? what channels and with the help of who?

Reply:
Yes, Andreea was going to work on a plan for this.